app/auth/views.py

```diff
@@ -0,0 +1,84 @@
+# /app/auth/views.py
+
+from . import auth_blueprint
+
+from flask.views import MethodView
+from flask import make_response, request, jsonify
+from app.models import User
+from app.views import db
+
+
+class RegistrationView(MethodView):
+    """This class registers a new user."""
+
+    def post(self):
+        # Query to see if the user already exists
+        user_email = User.query.filter_by(email=request.data["email"]).first()
+        user_username = User.query.filter_by(
+            username=request.data["username"]).first()
+        user = user_email or user_username
+
+        if not user:
+            try:
+                post_data = request.data
+                # Register user
+                username = post_data["username"]
+                email = post_data["email"]
+                password = post_data["password"]
```

Comment:
the assignment in lines 25-27 can be handled immediately after defining `post`. then use the variables in all your queries. 
Also it'd be better to use `request.data.get("email")` to avoid errors if one of the keys is missing in the request